hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -328,6 +330,20 @@ public class HoodieClusteringConfig extends HoodieConfig {
           + "Please exercise caution while setting this config, especially when clustering is done very frequently. This could lead to race condition in "
           + "rare scenarios, for example, when the clustering completes after instants are fetched but before rollback completed.");
 
+  public static final ConfigProperty<Boolean> BUCKET_CLUSTERING_MERGE_ENABLED = ConfigProperty
+      .key("hoodie.bucket.clustering.merge.enabled")
+      .defaultValue(true)
+      .markAdvanced()
+      .sinceVersion("0.14.0")
```

Comment:
Can we remove these options, if Spark supports it by default, we can ask the user to use spark for clustering execution instead.

Reply:
The options are used to control the resize plan instead of resize execution.

Reply:
Can flink handle the two options correctly？

Reply:
Flink write pipelines could not handle `hoodie.bucket.clustering.merge.enabled` correctly, so the value is set to `false` in flink engine automatically, and would check the resize plan does not include merge operations.
And Flink write pipelines could handle `hoodie.bucket.clustering.sort.enabled`. However it's time consuming, so the default value is set to `false` in flink engine automatically, and use could enable this if they need.

Reply:
Then just remove these two options and we can override these two actions in flink clustering plan strategy class.